easybuild/easyconfigs/p/pFUnit/pFUnit-4.2.0-gompi-2020b.eb

```diff
@@ -0,0 +1,65 @@
+easyblock = 'CMakeMake'
+
+name = 'pFUnit'
+version = '4.2.0'
+local_fargparse_major_minor_version = '1.1'
+local_gftl_major_minor_version = '1.3'
+local_gftl_shared_major_minor_version = '1.2'
+
+homepage = 'https://github.com/Goddard-Fortran-Ecosystem/pFUnit'
+description = """pFUnit is a unit testing framework enabling JUnit-like testing of serial
+ and MPI-parallel software written in Fortran."""
+
+toolchain = {'name': 'gompi', 'version': '2020b'}
+
+# Release does not include git submodules so do explicit clone instead
```

Comment:
The tarball in https://github.com/Goddard-Fortran-Ecosystem/pFUnit/releases/download/v4.2.0/pFUnit-4.2.0.tar does include the code for `extern/fArgParse` submodule. Could this be used instead of `git clone`?

Reply:
Yes please!

Reply:
Thanks for catching that, zipped tarballs just had the repo code and I didn't dig deeper